### PR TITLE
Editorial updates to Overview and Getting Started

### PR DESCRIPTION
**Pull request by SueChaplain**
Overview:

-  A little more introductory information for users
who might land directly on the docs without visiting the
Appsody home page.
- Remove the Docs section, which is repeated by the table of contents.
- Other edits and minor structural changes

Installation:

- Simplified slightly, removing pre-req section to leave Docker as a
single statement. Moved the macOS pre-reqs into the macOS section.
- Platform sections modified for consistency

Quick Start

- Remove Installation section and refer back to installation topic.
- Remove redundant "Using Appsody" H2 title.
- Make slightly more generic for all stacks, although still showing
nodejs-express as the example.

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

## Checklist

- [x] commit message follows [commit guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
- [x] website builds and runs in production - for information on how to test locally go [here](https://github.com/appsody/website/blob/master/DEVELOPMENT.md).

## Related Issues

#issue-number

### CODE REVIEW

**Comment by SueChaplain** [comment]
@neeraj-laad @tomleah  - please let me know if I've got any technical content wrong or you disagree with any changes.

In Quick Start where we provide the URL / port for `nodejs-express`, we tell users that other stacks might have different ports but now what they are. Think we should either tell them what they are or how to discover themselves.

**Comment by SueChaplain** [comment]
I found where to send users to look for the URL:port number, so updated my commit.

>In Quick Start where we provide the URL / port for nodejs-express, we tell users that other stacks might have different ports but not what they are. Think we should either tell them what they are or how to discover themselves.

File: content/docs/getting-started/quick-start.md [hunk 1]
```diff
@@ -2,43 +2,42 @@
...
+Great! Now the project is running in a docker container, and the container is linked to the project source code on your local system. For `nodejs-express`, navigate to <http://localhost:3000> to see the output. Other stacks might use a different URL, so check the [documentation](https://github.com/appsody/stacks/tree/master/incubator) for your chosen stack.
+
+Now let's try changing the code. Edit the file `app.js` to output something other than "Hello from Appsody!". When you save the file, Appsody picks up the change and automatically updates the container. Refresh <http://localhost:3000> to see the new message!
+
+You are ready to continue developing your application.
 
-You are ready to continue developing your application. To stop the container, press `Ctrl-C` in the terminal. To enable the debugger, restart the container with `appsody debug`.
+- To stop the container, press `Ctrl-C` in the terminal.
+- To enable the debugger, restart the container with `appsody debug`.
+- When you are ready to publish a production docker image, run `appsody build`.
```
Comment (tomleah) [inline content/docs/getting-started/quick-start.md]: We should reword this as `appsody build` does not publish an image, it just builds it ready to either publish or use.

File: content/docs/overview.md [hunk 2]
```diff
@@ -2,58 +2,42 @@
...
 
 ## How does it work?
 
-Appsody provides pre-configured application stacks, which use the well-known Dockerfile syntax to specify which language runtimes, frameworks, libraries and tools are included.
-
-Project templates build upon these stacks, and provide developers with a templatised application to bring into their IDE of choice and begin development.
+Appsody provides pre-configured application stacks, which use the well-known Dockerfile syntax to specify which language runtimes, frameworks, libraries, and tools are included. Project templates build upon these stacks, providing developers with a templatised application to bring into their IDE of choice and begin development.
```
Comment (tomleah) [inline content/docs/overview.md]: nit: `templatised` -> `templatized`

**Comment by tomleah** [comment]
Would appreciate another review from either @mhamwala or @neeraj-laad before merge. 
The content looks good. Feel free to counter my comments about style that we can resolve in another pass.

File: content/docs/getting-started/installation.md [hunk 3]
```diff
@@ -3,83 +3,57 @@ path: /docs/getting-started/installation
...
+1. If you don't have the Xcode command line tools installed on your system, install them by running `xcode-select --install`.
+2. If you don't have Homebrew installed on your system, install it by running `/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`.
+3. To install Appsody, run `brew install appsody/appsody/appsody`. This command creates a brew tap for the `appsody/appsody` repo and installs the `appsody` formula.
 
-Run this command: 
-```
-brew install appsody/appsody/appsody
-```
-The command will create a brew tap for the `appsody/appsody` repo, and install the `appsody` formula for you.
+That's all there is to it! You can now follow our [Quick Start](#quick-start) instructions to create your first containerized development environment with a running *Hello World!* application.
```
Comment (mhamwala) [inline content/docs/getting-started/installation.md]: should this link be pointing to `/docs/getting-started/quick-start`

File: content/docs/getting-started/installation.md [hunk 4]
```diff
@@ -3,83 +3,57 @@ path: /docs/getting-started/installation
...
+3. Install the package by running `sudo apt install -f <path>/appsody_v.r.m_amd64.deb`, where `<path>` is the fully qualified path to the package.
 
-To install Appsody on your system follow these steps:
-1) Download the latest **Debian install package** from the [Appsody releases page](https://github.com/appsody/appsody/releases). The file is named `appsody_v.r.m_amd64.deb`, where `v.r.m` indicates the release tag.
-2) Assuming, you downloaded the package into `~/Downloads`, run:
-```
-sudo apt install -f ~/Downloads/appsody_v.r.m_amd64.deb
-```
-Make sure you specify the path to the package file.
+That's all there is to it! You can now follow our [Quick Start](#quick-start) instructions to create your first containerized development environment with a running *Hello World!* application.
```
Comment (mhamwala) [inline content/docs/getting-started/installation.md]: same here

File: content/docs/getting-started/installation.md [hunk 5]
```diff
@@ -3,83 +3,57 @@ path: /docs/getting-started/installation
...
+3. Install the package by running `sudo yum install <path>/appsody-v.r.m-1.x86_64.rpm`, where `<path>` is the fully qualified path to the package.
 
-To install Appsody on your system follow these steps:
-1) Download the latest **RPM install package** from the [Appsody releases page](https://github.com/appsody/appsody/releases). The file is named `appsody-v.r.m_1.x86_64.rpm`, where `v.r.m` indicates the release tag.
-2) Assuming, you downloaded the package into `~/Downloads`, run:
-```
-sudo yum install ~/Downloads/appsody-v.r.m_1.x86_64.rpm
-```
-Make sure you specify the path to the package file.
+That's all there is to it! You can now follow our [Quick Start](#quick-start) instructions to create your first containerized development environment with a running *Hello World!* application.
```
Comment (mhamwala) [inline content/docs/getting-started/installation.md]: same here

File: content/docs/getting-started/quick-start.md (hunk 1 above)
Comment (SueChaplain) [reply content/docs/getting-started/quick-start.md]: Fixing.

File: content/docs/getting-started/installation.md [hunk 6]
```diff
@@ -3,83 +3,57 @@ path: /docs/getting-started/installation
...
-xcode-select --install
-brew install appsody
-```
+Follow these steps:
 
-## Installing on macOS
-Ensure the [prerequisites](#Prerequisites) are met.
+1. If you don't have the Xcode command line tools installed on your system, install them by running `xcode-select --install`.
+2. If you don't have Homebrew installed on your system, install it by running `/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`.
+3. To install Appsody, run `brew install appsody/appsody/appsody`. This command creates a brew tap for the `appsody/appsody` repo and installs the `appsody` formula.
```
[the remark this answers is not in the document]
Comment (SueChaplain) [reply content/docs/getting-started/installation.md]: Yes, looks better. I did the same for other platforms to balance things.

File: content/docs/overview.md (hunk 2 above)
Comment (SueChaplain) [reply content/docs/overview.md]: I changed it but that wasn't a word I used - it was already there ..... infact I don't like made up words like that. We should probably say something different. Let's let it go for now.

File: content/docs/getting-started/installation.md (hunk 3 above)
Comment (SueChaplain) [reply content/docs/getting-started/installation.md]: Yes! Good spot. Changed.

File: content/docs/getting-started/installation.md (hunk 4 above)
Comment (SueChaplain) [reply content/docs/getting-started/installation.md]: changed

File: content/docs/getting-started/installation.md (hunk 5 above)
Comment (SueChaplain) [reply content/docs/getting-started/installation.md]: changed
